roles/ps3walk.role

```diff
@@ -18,7 +18,7 @@ nuclear_role(
   actuation::FootController
   # Skills
   skill::Walk
-  skill::SplineKick
+  skill::ScriptKick
```

Comment:
how come this is changed

Reply:
It was a quick fix tom did because the kick was unstable when we tested it. I'll check this is correct as well.

Reply:
Did the script kick work? I didn't think they were tuned

Reply:
It did at the time, but I'll have a look at changing it back if it'll work.